server/handle_policy_eval.go

```diff
@@ -176,3 +251,37 @@ func PolicyEval(policyRego string, policyQuery string, input interface{}) (decis
 
 	return true, nil
 }
+
+func PolicyEvalGetRawResult(policyRego string, input interface{}) (results rego.ResultSet, err error) {
```

Comment:
we can extract a common function from the  `PolicyEvalGetRawResult` and the `PolicyEval`.

Reply:
Done.